static/app/components/nav/secondary.tsx

```diff
@@ -0,0 +1,141 @@
+import type {ReactNode} from 'react';
+import {createPortal} from 'react-dom';
+import styled from '@emotion/styled';
+
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import Link, {type LinkProps} from 'sentry/components/links/link';
+import {useNavContext} from 'sentry/components/nav/context';
+import {space} from 'sentry/styles/space';
+import {useLocation} from 'sentry/utils/useLocation';
+
+type SecondaryNavProps = {
+  children: ReactNode;
+};
+
+interface SecondaryNavItemProps extends Omit<LinkProps, 'ref'> {
+  children: ReactNode;
+  to: string;
+  isActive?: boolean;
+}
+
+export function SecondaryNav({children}: SecondaryNavProps) {
+  const {secondaryNavEl} = useNavContext();
+
+  if (!secondaryNavEl) {
+    return null;
+  }
+
+  return createPortal(children, secondaryNavEl);
+}
+
+SecondaryNav.Body = function SecondaryNavBody({children}: {children: ReactNode}) {
+  return <Body>{children}</Body>;
+};
```

Comment:
adding functions to a function object seems cursed

Reply:
that's javascript for ya